Apps/FlightSoftware/fprime/CubeRover/WatchDogInterface/WatchDogInterface.hpp

```diff
@@ -17,55 +17,69 @@
 #define WatchDogInterface_HPP
 
 #include "CubeRover/WatchDogInterface/WatchDogInterfaceComponentAc.hpp"
+#include <CubeRover/WatchDogInterface/WatchDogRxTask.hpp>
 #include "Include/FswPacket.hpp"
 
 #include "lin.h"
 #include "adc.h"
 #include "sci.h"
 
 // The pin number for the deploment pin 2
-const U8 deploy_bit = 4;
+static const U8 deploy_bit = 4;
 // The number of thermistors on the SBC
-const U8 number_thermistors = 6;
+static const U8 number_thermistors = 6;
 // Default size of zero sent to watchdog
-const U16 zero_size = 0x0000;
+static const U16 zero_size = 0x0000;
 // Minimum size that should be received back from watchdog when receiving
-const U8 min_receive_size = 8;
+static const U8 min_receive_size = 8;
 // Magic Value first sent in the header between watchdog and hercules communication
-const U32 header_magic = 0x21B00B;
+static const U32 header_magic = 0x21B00B;
+
+// This controls how long we'll wait for a response before allowing the next message of the same type
+// to be sent to the MSP430 watchdog
+static const uint32_t COMMAND_TIMEOUT_MILLISECONDS = 2000;
+
+// These three parameters control the setup of the task that handles data received from the MSP430 watchdog.
+static const NATIVE_INT_TYPE WATCH_DOG_INTERFACE_RX_TASK_PRIORITY = 16;
+static const NATIVE_INT_TYPE WATCH_DOG_INTERFACE_RX_TASK_STACK_SIZE = 256;
+static const NATIVE_INT_TYPE WATCH_DOG_INTERFACE_RX_TASK_CPU_AFFINITY = -1;
```

Comment:
These are very likely to change. I just picked these arbitrarily, and we'll need to figure out what they actually need to be